src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -876,4 +877,20 @@ public void setAudioProperties(AudioProperties audioProperties) {
 		}
 		this.audioProperties = audioProperties;
 	}
+
+	/**
+	 *
+	 * @return user rating (0 - 5 stars)
+	 */
+	public Integer getRating() {
```

Comment:
I think @SurfaceS suggested to add rating attribute to files, because other types except audio can then also be rated. Basically a good idea. But you might be right, if a file (maybe a cue file) has multiple tracks, this logic fails. I think I'll move the rating column back to Audiotack table.

Reply:
I see. I didn't realize that was already discussed. It's ok either way for me